docs/architecture/microservices/multi-container-microservice-net-applications/microservice-application-design.md

```diff
@@ -113,7 +113,7 @@ A microservice based solution like this has many benefits:
 
 A microservice based solution like this also has some drawbacks:
 
-**Distributed application**. Distributing the application adds complexity for developers when they are designing and building the services. For example, developers must implement interservice communication using protocols like HTTP or AMPQ, which adds complexity for testing and exception handling. It also adds latency to the system.
+**Distributed application**. Distributing the application adds complexity for developers when they are designing and building the services. For example, developers must implement inter-service communication using protocols like HTTP or AMPQ, which adds complexity for testing and exception handling. It also adds latency to the system.
```

Comment:
shouldn't this be intra-service? @rpetrusha can you help here?

```suggestion
**Distributed application**. Distributing the application adds complexity for developers when they are designing and building the services. For example, developers must implement intra-service communication using protocols like HTTP or AMPQ, which adds complexity for testing and exception handling. It also adds latency to the system.
```

Reply:
In this context it means communication between separate and independent services, I think that should be "inter-", right?